projects/rero/ng-core/src/lib/record/editor/type/custom-select/custom-select.component.ts

```diff
@@ -140,6 +140,19 @@ export class CustomSelectFieldComponent extends FieldType implements OnDestroy,
    * @returns The selected values as string.
    */
   get selectedValuesAsString(): string {
+    if (Array.isArray(this.formControl.value)) {
+      // Keeps selected items in order
+      const data = [];
+      let selectedOption: SelectOption;
+      const selectedValue = this.formControl.value.filter((v: string) => v !== undefined);
```

Comment:
This can be done in the foreach to avoid 2 iterations over the list.

Reply:
I wanted to avoid an "if" in the foreach. So I filtered this list to eliminate the "undefined" ones. But if you want, I can put it in the loop?